axelor-business-support/src/main/resources/views/TeamTask.xml

```diff
@@ -23,6 +23,13 @@
 		<field name="projectCategory"/>
 		<field name="targetVersion"/>
 	</grid>
+	
+    <grid model="com.axelor.apps.project.db.ProjectPlanningTime" title="Time spent" name="team-task-client-project-planning-time-spent-grid">
```

Comment:
grid and form name for client should be project-planning-time-spent-client-grid/form .